rbac/management/tenant_service/v2.py

```diff
@@ -103,6 +104,94 @@ def update_user(
 
         return bootstrapped_tenant
 
+    def _bootstrap_tenants(self, tenants: list[Tenant]) -> list[BootstrappedTenant]:
+        # Set up workspace hierarchy for Tenant
+        workspaces = []
+        relationships = []
+        mappings_to_create = []
+        default_workspace_uuids = []
+        tenant_mappings = {}
+        for tenant in tenants:
+            root_workspace_uuid = uuid4()
+            default_workspace_uuid = uuid4()
+            default_workspace_uuids.append(default_workspace_uuid)
+            workspaces.append(
+                Workspace(uuid=root_workspace_uuid, tenant=tenant, type=Workspace.Types.ROOT, name="Root Workspace")
+            )
+            workspaces.append(
+                Workspace(
+                    uuid=default_workspace_uuid, tenant=tenant, type=Workspace.Types.DEFAULT, name="Default Workspace"
+                )
+            )
+            # Existing tenant might have mapping or not
+            if not hasattr(tenant, "tenantmapping"):
+                mappings_to_create.append(TenantMapping(tenant=tenant))
+            else:
+                tenant_mappings[tenant.id] = tenant.tenantmapping
+            tenant_id = f"{self._user_domain}/{tenant.org_id}"
+            relationships = [
+                create_relationship(
+                    ("rbac", "workspace"),
+                    str(default_workspace_uuid),
+                    ("rbac", "workspace"),
+                    str(root_workspace_uuid),
+                    "parent",
+                ),
+                create_relationship(
+                    ("rbac", "workspace"), str(root_workspace_uuid), ("rbac", "tenant"), tenant_id, "parent"
+                ),
+            ]
+
+            # Include platform for tenant
+            relationships.append(
+                create_relationship(("rbac", "tenant"), tenant_id, ("rbac", "platform"), settings.ENV_NAME, "platform")
+            )
+
+        # Ignore those already created
+        Workspace.objects.bulk_create(workspaces, ignore_conflicts=True)
```

Comment:
Are you ignoring conflicts so that we can re-run the import? In that case maybe we should actually use `update_conflicts`.

Reply:
Yea, I was avoiding conflicts of the created workspaces for the bootstrapped tenants. Now I refactored the code a bit to check the existing ones to use their uuid